src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -1295,3 +1295,30 @@ def get_user_list(self, zoneName=None):
         multisite_instance = RgwMultisite()
         result = multisite_instance.get_user_list(zoneName)
         return result
+
+@APIRouter('/rgw/Topics', Scope.RGW)
+class RgwTopicManagement(RESTController):
+    @RESTController.Collection(method='POST', path='/createTopic')
+    @allow_empty_body
+    def create_topic(self, daemon_name=None, owner=None,topic_name:str =None,push_endpoint: Optional[str]=None,opaqueData: Optional[str]=None, persistent: Optional[str]=None,time_to_live: Optional[str]=None,max_retries: Optional[str]=None,retry_sleep_duration: Optional[str]=None,policy: Optional[str]=None):
```

Comment:
So this would  be POST if we just use `create`.
Thats how we define in general for dashboard:

```
POST /api/rgw/topic
GET /api/rgw/topic
DELETE /api/rgw/topic
```

```suggestion
    @allow_empty_body
    def create(
    self, 
    topic_name: Optional[str] = None,
    daemon_name: Optional[str] = None, 
    owner: Optional[str] = None,
    push_endpoint: Optional[str] = None,
    opaque_data: Optional[str] = None,
    persistent: Optional[bool] = None,
    time_to_live: Optional[int] = None,
    max_retries: Optional[int] = None,
    retry_sleep_duration: Optional[float] = None,
    policy: Optional[str] = None
) -> None:
```

Reply:
done